src/VolleyManagement.Backend/VolleyManagement.UI/Areas/Mvc/Controllers/Domain/TeamsController.cs

```diff
@@ -171,21 +180,38 @@ public JsonResult Edit(TeamViewModel teamViewModel)
             {
                 try
                 {
-                    var roster = teamViewModel.Roster.Select(t => t.ToDomain()).ToList();
+                    if (teamViewModel.AddedPlayers.Count > 0)
+                    {
+                        var playersIdsToAddToTeam = teamViewModel.AddedPlayers
+                            .Where(x => x.Id > 0)
+                            .Select(x => new PlayerId(x.Id))
+                            .ToList();
+
+                        var playersToAddToTeam = teamViewModel.AddedPlayers
+                            .Where(x => x.Id == 0)
+                            .Select(x => x.ToCreatePlayerDto())
+                            .ToList();
+
+                        if (playersToAddToTeam.Count != 0)
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Refactor this code to not nest more than 3 control flow statements. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS134)